packages/cctp/schema.graphql

```diff
@@ -0,0 +1,25 @@
+type MessageReceived @entity(immutable: true) {
+  id: Bytes!
+  caller: Bytes! # address
+  sourceDomain: BigInt! # uint32
+  nonce: BigInt! # uint64
+  sender: Bytes! # bytes32
+  messageBody: Bytes! # bytes
+  blockNumber: BigInt!
+  blockTimestamp: BigInt!
+  transactionHash: Bytes!
+}
+
+type MessageSent @entity(immutable: false) {
```

Comment:
Why is `immutable: false`? Are you modifying any of the entity's fields? (same for L2 version)

Reply:
It was the case previously, but not anymore. I reverted it

Reply:
Actually, we do we need if `replaceMessage` was called